param/optical_flow_params_1.0.yaml

```diff
@@ -6,6 +6,11 @@ optical_flow_estimator:
     # Minimum altitude to run the detector
     min_estimation_altitude: 0.3
 
+    # Camera vertical threshold
+    # Minimum dot product between camera forward vector and down vector
+    # required for optical flow to run
+    camera_vertical_threshold: 0.9
```

Comment:
Can we do this as an angle in radians? This is hard for a human to understand.

Reply:
k